### PR TITLE
improve SK translations

### PR DESCRIPTION
**Pull request by radoslavzeman**
### What does it do?

Improves and adds more slovak (SK) translations for admin and upload plugin.

### Why is it needed?

For slovak users :)

### How to test it?

Provide information about the environment and the path to verify the behaviour.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

---> question in the comment

### CODE REVIEW

File: packages/core/admin/admin/src/translations/sk.json [hunk 1]
```diff
@@ -92,51 +164,72 @@
...
   "Settings.webhooks.list.empty.description": "Pridajte Váš prvý webhook do zoznamu.",
   "Settings.webhooks.list.empty.link": "Pozrieť si našu dokumentáciu",
   "Settings.webhooks.list.empty.title": "Zatiaľ neexistujú žiadne Webhooky",
+  "Settings.webhooks.list.th.actions": "akcie",
+  "Settings.webhooks.list.th.status": "stav",
   "Settings.webhooks.singular": "Webhook",
   "Settings.webhooks.title": "Webhooky",
-  "Settings.webhooks.trigger": "Trigger",
-  "Settings.webhooks.trigger.cancel": "Zrušiť Trigger",
+  "Settings.webhooks.to.delete": "{webhooksToDeleteLength, plural, one {# vybraný záznam} few {# vybrané záznamy} other {# vybraných záznamov}}",
```
Comment (radoslavzeman) [inline packages/core/admin/admin/src/translations/sk.json:194]: ### Question:
How do we specify plural translations when number is between 2 and 4? I used keyword `few` as I know it from other translation libraries, but I'm not sure, if it is supported here.

The way how it works in slovak language is, that there exists a form of noun or adjective for:
`1` entity
`2 to 4` entities
`5 and more` entities
(`0` is the same as `5 and more`)

**Review by ronronscelestes — APPROVED** [review]
LGTM, thank you @radoslavzeman for this nice addition :)

File: packages/core/admin/admin/src/translations/sk.json (hunk 1 above)
Comment (ronronscelestes) [reply packages/core/admin/admin/src/translations/sk.json:194]: hi! 
I checked the [documentation](https://formatjs.io/docs/core-concepts/icu-syntax/#selectordinal-format) of the library we use and it supports few, and it adapts the range to the language - I also tested your branch just to be sure:
image
image
image
image

I didn't know about those specificities, glad to know more now 😄

Comment (radoslavzeman) [reply packages/core/admin/admin/src/translations/sk.json:194]: Thanks!

**Pull request merged**